Kitodo-API/src/main/java/org/kitodo/api/dataformat/MediaUnit.java

```diff
@@ -37,6 +49,23 @@
      */
     private String orderlabel;
 
+    /**
+     * The subordinate media units of this media unit, which form the media unit
+     * tree. The order of the substructures is subordinate media units by the
+     * order of the {@code <div>} elements in the
+     * {@code <structMap TYPE="PHYSICAL">} in the METS file.
+     */
+    private List<MediaUnit> subordinateUnits = new LinkedList<>();
+
+    /**
+     * Returns the subordinate media units associated with this media unit.
+     * 
+     * @return the subordinate media units
+     */
+    public List<MediaUnit> getChildren() {
```

Comment:
Wouldn't it be more intuitive to either call this method `getSubordinateUnits` or the corresponding field `children`?